config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: registry.hub.docker.com/tuentitech/secrets-manager:v2.1.0
+      - image: registry.hub.docker.com/tuentitech/secrets-manager:v2.1.1
```

Comment:
Don't change, 2.1.0 will be the next version to be released.

Reply:
:+1: thanks!

Reply:
@Wiston999 removed version bump from properties and image patch files.